packages/strapi-tiptap-editor/admin/src/context/productPrice/state.tsx

```diff
@@ -28,7 +28,7 @@ const State = (props: any) => {
                     data {
                       attributes {
                         title
-                        price {
+                        price(pagination: {limit: 100}) {
```

Comment:
Kunnen we met zekerheid zeggen dat het aantal prijzen altijd maximaal 100 is? Is het slimmer om pagination in zijn volledigheid gewoon uit te zetten voor deze query?

`price(pagination : { start : 0 , limit : -1 }) {...`

Ik lees wel dat er überhaupt een limiet van 100 in de graphql config voor strapi zit. We zouden even moeten onderzoeken of er gekke gevallen zijn waar we meer dan dat nodig hebben.